doc/source/install.rst

```diff
@@ -185,7 +185,7 @@ For Anaconda users experiencing problems during installation of ixmp, check that
     C:\[YOUR ANACONDA LOCATION]\Anaconda3\Scripts;
     C:\[YOUR ANACONDA LOCATION]\Anaconda3\Library\bin;
 
-.. _`installing MESSAGEix`: https://message.iiasa.ac.at/en/latest/getting_started.html
+.. _`installing MESSAGEix`: https://docs.messageix.org/en/stable/getting_started.html
```

Comment:
Please keep this at `latest`. We often update & improve the install docs between releases and want to point to the latest.